MyMovie/TestApp/permissions.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import permissions
+from django.http import Http404
+from TestApp.models import *
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class UserPermission(BasePermission):
+    message = "You cant rate it "
+
+    def has_permission(self, request, view):
+        movie = Movie.objects.get(pk=request.data['movie'])
```

Comment:
Please add a try/catch statement here. There's a chance the movie might not exist.